mysql/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM mysql:5.7
+
+# Automatically start the SSH agent when a new session is created
+RUN echo 'max_allowed_packet=256M' >> /etc/mysql/conf.d/docker.cnf
```

Comment:
Could you add a newline at the end of file?

Reply:
Done :)